### PR TITLE
[WIP] 6979 content hub page

### PR DESCRIPTION
**Pull request by tomas-stefano**

### CODE REVIEW

**Comment by rnalexander** [comment]
:+1:

File: app/assets/stylesheets/comfortable_mexican_sofa/admin/components/_content_hub_module.scss [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    @include column(6);
+  }
+
+  .content-hub-module__hero-image {
+    @include column(6);
+  }
+
+  .content-hub-module__hero-title {
+
+  }
```
Comment (munckymagik) [inline app/assets/stylesheets/comfortable_mexican_sofa/admin/components/_content_hub_module.scss]: Is there a plan to use these later? Or are they intentionally left blank?

**Comment by munckymagik** [comment]
spec/javascripts/tests/PageFormType_spec.js is empty. Is this a WIP PR?

File: app/assets/stylesheets/comfortable_mexican_sofa/admin/components/_content_hub_module.scss (hunk 1 above)
Comment (johnplayer1982) [inline app/assets/stylesheets/comfortable_mexican_sofa/admin/components/_content_hub_module.scss]: Hey @munckymagik yes these will be used later

**Comment by tomas-stefano** [comment]
Closing since we're not touch on this for some time.

**Pull request closed without merging**